src/main/java/com/todoist/pojo/Item.kt

```diff
@@ -10,7 +10,7 @@ open class Item<D : Due> @JvmOverloads constructor(
     due: D?,
     open var sectionId: Long?,
     open var parentId: Long?,
-    open var childOrder: Int = MIN_CHILD_ORDER,
+    open var childOrder: Int = DEFAULT_CHILD_ORDER,
     open var dayOrder: Int = DEF_DAY_ORDER,
```

Comment:
Could we also rename this constant for consistency please?

```suggestion
    open var dayOrder: Int = DEFAULT_DAY_ORDER,
```